src/libs/actions/Session/index.js

```diff
@@ -72,7 +72,6 @@ function signOut() {
         partnerUserID: lodashGet(credentials, 'autoGeneratedLogin', ''),
         partnerName: CONFIG.EXPENSIFY.PARTNER_NAME,
         partnerPassword: CONFIG.EXPENSIFY.PARTNER_PASSWORD,
-        shouldRetry: false,
```

Comment:
That was a good question. 
I just tested it and we should be fine with this since it's the log out command. 
What happens is:
- We call Logout
- We clear the cache

If logout fails, when we go to retry, there are no requests here: https://github.com/Expensify/App/blob/9a63feb9f28719665d319a38f70041d865c71fd4/src/libs/Network/SequentialQueue.js#L34

So we just return and go back to the login screen, so it works as expected.

Reply:
I don't think this param does anything when called inside the `SequentialQueue`. I think it will be retried regardless. We can test this out though by expiring the authToken (mess it up somehow) then try to `SignOut`. We should be able to see a request to `LogOut` then a call to `Authenticate` and another call to `LogOut`.

Reply:
As I commented above this won't be retried because we clear the queue if it fails which still has the expected behavior so we're fine with this command

Reply:
Ok not a blocker just wanted to call out that the param is currently not doing anything 😄